README.md

```diff
@@ -127,3 +127,11 @@ $ cargo benchcmp old new --regressions --threshold 2
  name                                full:: ns/iter     full_overlap:: ns/iter  diff ns/iter  diff %  speedup
  ac_ten_one_prefix_byte_every_match  27,424 (364 MB/s)  28,046 (356 MB/s)                622   2.27%   x 0.98
 ```
+
+You can also concatenate measurements of the same version of benchmarks, to reduce noise.
+```
+$ cargo bench >> control
+```
+When comparing a file with multiple measurements in it,
+benchcmp will pick the quickest or most stable measurement of each run,
```

Comment:
Could you say, "will pick the quickest measurement of each run." It could otherwise be interpreted as "benchcmp will either choose the quickest measurement, or select the most stable measurement depending on heuristics." Of course, I realize you meant "or" as "or in other words," but I'd prefer to just be a bit more direct here.

Reply:
I didn't actually mean "in other words", but rather that if two measurements are equal, the lesser variance is deemed better. I'll add that, or do you think that's too much detail?

Also, I forgot to mention I don't really know what a benchmark's optional throughput is. If one can measure a higher throughput over a longer amount of time, that seems a way better result. But I hope that throughput scales inversely with ns.

Reply:
Ah yeah, I would add that detail then, yes.

> Also, I forgot to mention I don't really know what a benchmark's optional throughput is. If one can measure a higher throughput over a longer amount of time, that seems a way better result. But I hope that throughput scales inversely with ns.

It does, yes. Throughput is reported if you tell the benchmark harness how big your input is.